presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/ClickHouseConfig.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.clickhouse;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+import com.facebook.airlift.configuration.ConfigSecuritySensitive;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+public class ClickHouseConfig
+{
+    private String connectionUrl;
+    private String connectionUser;
+    private String connectionPassword;
+    private String userCredentialName;
+    private String passwordCredentialName;
+    private boolean caseInsensitiveNameMatching;
+    private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
+    private boolean mapStringAsVarchar;
+
+    @NotNull
+    public String getConnectionUrl()
+    {
+        return connectionUrl;
+    }
+
+    @Config("connection-url")
+    public ClickHouseConfig setConnectionUrl(String connectionUrl)
+    {
+        this.connectionUrl = connectionUrl;
+        return this;
+    }
+
+    @Nullable
+    public String getConnectionUser()
+    {
+        return connectionUser;
+    }
+
+    @Config("connection-user")
```

Comment:
`clickhouse.connection-user` same for all configurations, let's add connector prefix